From e9d62c4fbc1164be13752a08fa40340aef4dd8d0 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:36 -0400
Subject: [redhat] kabi: Adding symbol time_to_tm (kernel/time/timeconv.c)

Message-id: <b2f4f04a2ec3238444e0cabf5d852916410fb06f.1410510773.git.poros@redhat.com>
Patchwork-id: 96059
O-Subject: [PATCH RHEL7.1 216/233] kabi: Adding symbol time_to_tm (kernel/time/timeconv.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    LLT logs output of do_gettimeofday using this. The resultant is used to tage LLT buffers.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/time_to_tm
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/time_to_tm
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/time_to_tm

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/time_to_tm b/redhat/kabi/kabi-module/kabi_ppc64/time_to_tm
new file mode 100644
index 0000000..6976fef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/time_to_tm
@@ -0,0 +1,2 @@
+#1-
+0x92a9c60c time_to_tm vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/time_to_tm b/redhat/kabi/kabi-module/kabi_s390x/time_to_tm
new file mode 100644
index 0000000..6976fef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/time_to_tm
@@ -0,0 +1,2 @@
+#1-
+0x92a9c60c time_to_tm vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/time_to_tm b/redhat/kabi/kabi-module/kabi_x86_64/time_to_tm
new file mode 100644
index 0000000..6976fef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/time_to_tm
@@ -0,0 +1,2 @@
+#1-
+0x92a9c60c time_to_tm vmlinux EXPORT_SYMBOL
-- 
1.7.1